modular/gater/object_handler.go

```diff
@@ -140,12 +140,24 @@ func (g *GateModular) getObjectHandler(w http.ResponseWriter, r *http.Request) {
 		}
 		log.CtxDebugw(reqCtx.Context(), reqCtx.String())
 	}()
-
 	reqCtx, err = NewRequestContext(r)
 	if err != nil {
-		return
+		// check the object's visibility type whether equal to public read
+		authorized, err = g.baseApp.Consensus().VerifyGetObjectPermission(reqCtx.Context(), "",
+			reqCtx.bucketName, reqCtx.objectName)
+		if err != nil {
+			log.CtxErrorw(reqCtx.Context(), "failed to verify authorize for getting public object", "error", err)
```

Comment:
`failed to verify authorize` is fixed as `failed to verify authorization` is better. authorize is verb.

Reply:
next pr

Reply:
@will-2012 is doing log tidy in netx pr